aroquev/src/main/webapp/index.html

```diff
@@ -5,9 +5,21 @@
   <meta charset="UTF-8">
   <title>My Portfolio</title>
   <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP&display=swap" rel="stylesheet">
+  <link rel="icon" 
+      type="image/png" 
+      href="images/noogler-hat.png">
 </head>
 
 <body onload="getServletComments()">
+  <div class="topnav">
+    <a class="active" href="#">Home</a>
+    <a href="#about-me">About Me</a>
```

Comment:
You could make these into separate pages (about.html, projects.html, etc.). You aren't constrained to having every section inside index.html.

Reply:
True, it's starting to look a bit too cramped.